app/models/hud_reports/generator_base.rb

```diff
@@ -49,6 +50,18 @@ def client_scope(start_date: @report.start_date, end_date: @report.end_date)
       scope = scope.merge(GrdaWarehouse::ServiceHistoryEnrollment.in_coc(coc_code: @report.coc_codes)) if @report.coc_codes.present?
       scope = scope.merge(GrdaWarehouse::ServiceHistoryEnrollment.in_project(@report.project_ids)) if @report.project_ids.present?
 
+      @filter = self.class.filter_class.new(user_id: @report.user_id).update(@report.options)
+      she_scope = GrdaWarehouse::ServiceHistoryEnrollment.all
```

Comment:
Pull `in_coc` and `in_project` into this scope instead of the merge, just for hygiene?